iConverter/Services/HistoryDataStore/HistoryDataStore.swift

```diff
@@ -10,29 +10,16 @@ import RxRelay
 import NSObject_Rx
 
 protocol HistoryDataStoreProtocol {
-    var updateHistory: PublishRelay<Void> { get }
-    var history: BehaviorRelay<[Transaction]> { get }
-
     func save(transaction: Transaction)
+    func loadTransactions() -> [Transaction]
 }
 
-final class HistoryDataStore: HistoryDataStoreProtocol, HasDisposeBag {
+final class HistoryDataStore: HistoryDataStoreProtocol {
```

Comment:
Again, repositories are just for read/writing entities. We keep this to have just those responsibilities. In case changes here needs to be observed in multiple places, we can use notification, in the app, we use `EventBusManager` then just query the updated data. Please see `TransferViewController.viewDidLoad`